File: MCRRTImp/src/test/java/lab/mars/MCRRTImp/Vector2BasedImp/TestBound.java
package lab.mars.MCRRTImp.Vector2BasedImp;

import infrastructure.Vector2BasedImp.Bound;
import infrastructure.Vector2BasedImp.Vector2;
import org.junit.Test;

public class TestBound {

    @Test
    public void testContains() {
        Bound bound = new Bound(100, 100);
        assert bound.contains(new Vector2(100.1, 100));
    }
}

File: MCRRTImp/src/test/kotlin/lab/mars/MCRRTImp/Vector2BasedImp/TestBound.kt
package lab.mars.MCRRTImp.Vector2BasedImp

import infrastructure.Vector2BasedImp.Bound
import infrastructure.Vector2BasedImp.Vector2
import org.junit.Test

class TestBound {

    @Test
    fun testContains() {
        val bound = Bound(100.0, 100.0)
        assert(bound.contains(Vector2(100.1, 100.0)))
    }
}
